app/views/qsets/show.html.erb

```diff
@@ -1,5 +1,8 @@
 <div class="container-fluid">
   <%= render "qset_context" %>
+  <div id="generate-question-button-filter">
```

Comment:
I think it would be best if the `#generate-question-button-filter` and the `#all-mine-filter-container` were contained in their own bootstrap `.row` -- that would probably avoid the idiosyncratic `margin` style we apply to it right now. It also follows bootstrap's recommended principles, and if we stray from those we'll get into more trouble down the line probably.

When I look at the layout, it looks like:
1. row for qset context
2. row for generate question button + all-mine-other filter
3. row to encapsulate all the stuff below those two rows above (i.e. the subset list and the question list)

We're missing row 2 above. It would make sense for it to have one `.col-sm-3` column and one `.col-sm-9` column, to match the row below, probably.